lib/logstream/logstream.go

```diff
@@ -784,12 +809,8 @@ func (g *ContainerLogConsumer) Accept(l tc.Log) {
 		return
 	}
 
-	if g.isDone {
-		g.ls.log.Error().
-			Str("Test", g.ls.testName).
-			Str("Container", g.name).
-			Str("Log", string(l.Content)).
-			Msg("Consumer has finished, but you are still trying to accept logs. This should never happen")
```

Comment:
how will you know now that you're tying to send the logs, even though you should not?

Reply:
@Tofel to be honest I don't care about it here. I think this can still happen until we migrate to "Passing the LogConsumers in the ContainerRequest" approach https://golang.testcontainers.org/features/follow_logs/?utm_source=chatgpt.com#passing-the-logconsumers-in-the-containerrequest. This will be a separate ticket.